src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -188,6 +188,10 @@ export default function transformForSubmit(formConfig, form) {
     dataPostTransform.veteran.address.postalCode || '';
   delete dataPostTransform.veteran.address.postalCode;
 
+  dataPostTransform.certifierRole = transformedData.certifierRole;
```

Comment:
This line will disappear as a change once https://github.com/department-of-veterans-affairs/vets-website/pull/30571 is merged